modweaver.testmod/ModMain.cs

```diff
@@ -1,51 +1,65 @@
-using System;
+using System.Collections.Generic;
 using System.IO;
 using HarmonyLib;
 using modweaver.core;
+using UnityEngine;
+using modweaver.api.API;
 
 
 namespace modweaver.testmod {
     [ModMainClass]
     // ReSharper disable once UnusedType.Global (because it is used!!)
     public class ModMain : Mod {
+
+        public static List<global::Weapon> newWeapons;
+        CustomWeapon weapon1;
+
         public override void Init() {
-            //test logger
```

Comment:
Why did you delete the logging tests? It's important to test all aspects of the api in the testmod.